provers/cairo/src/main.rs

```diff
@@ -198,6 +201,7 @@ fn main() {
                 eprintln!("\nYou are trying to prove a non compiled Cairo program. Please compile it before sending it to the prover.\n");
                 return;
             }
+            println!("{:?}", args.program_path);
```

Comment:
seems like a debug print. Should it be removed?

Reply:
Yes, deleting it.